### PR TITLE
update for multitoken

### PR DESCRIPTION
**Pull request by yito88**

### CODE REVIEW

File: packages/specs/pages/interoperability/ibc.md [hunk 1]
```diff
@@ -1,71 +1,75 @@
...
+A transaction escrowing/unescrowing a token changes the escrow account's balance of the token. The key is `#Multitoken/{token_addr}/balance/#Ibc`. A transaction burning a token decreases the minted balance of the token. The key is `#Multitoken/{token_addr}/balance/#Mint`. A transaction minting a token increases the minted balance of the token. The key is `#Multitoken/{token_addr}/balance/#Mint`. `#Multitoken`, `#Mint` and `#Ibc` are addresses of [`InternalAddress`](https://github.com/anoma/namada/blob/a5bad396992e5f66351088bde3bec73d83e769ba/core/src/types/address.rs#L473) and actually they are encoded in the storage key. Also, the multitoken validity predicate will be triggered and check the balance changes.
```
Comment (cwgoes) [inline packages/specs/pages/interoperability/ibc.md:28]: These keys for minted/burned tokens seem wrong?

File: packages/specs/pages/base-ledger/fungible-token.md [hunk 2]
```diff
@@ -2,28 +2,10 @@
...
+A token balance is stored with a storage key. The token balance key should be `#Multitoken/{token_addr}/balance/{owner_addr}`. These keys can be made with [token functions](https://github.com/anoma/namada/blob/5da82f093f10c0381865accba99f60c557360c51/core/src/types/token.rs).
 
-We can have multitoken balances with the same token and the same owner by `{sub_prefix}`, e.g. a token balance received over IBC is managed in `{token_addr}/ibc/{ibc_token_hash}/balance/{receiver_addr}`. It is distinguished from the receiver's original balance in `{token_addr}/balance/{receiver_addr}` to know which chain the token was transferred from.
+We can have multitoken balances for the same owner by `{token_addr}`, e.g. a token received over IBC is managed in `#Multitoken/{ibc_token}/balance/{receiver_addr}`. It is distinguished from the receiver's original balance in `#Multitoken/{token_addr}/balance/{receiver_addr}` to know which chain the token was transferred from.
```
Comment (cwgoes) [inline packages/specs/pages/base-ledger/fungible-token.md:7]: We should describe how `ibc_token` is derived.

File: packages/specs/pages/interoperability/ibc.md [hunk 3]
```diff
@@ -1,71 +1,75 @@
...
+A transaction escrowing/unescrowing a token changes the escrow account's balance of the token. The key is `#Multitoken/{token_addr}/balance/#Ibc`. A transaction burning a token decreases the minted balance of the token. The key is `#Multitoken/{token_addr}/balance/#Minted`. A transaction minting a token increases the minted balance of the token. The key is `#Multitoken/{token_addr}/balance/#Minted`. `#Multitoken`, `#Minted` and `#Ibc` are addresses of [`InternalAddress`](https://github.com/anoma/namada/blob/a5bad396992e5f66351088bde3bec73d83e769ba/core/src/types/address.rs#L473) and actually they are encoded in the storage key. Also, the multitoken validity predicate will be triggered and check the balance changes.
```
Comment (cwgoes) [inline packages/specs/pages/interoperability/ibc.md:28]: Wait, why does `minted` need to be an internal address? Can it just be a sentinel value?

File: packages/specs/pages/base-ledger/fungible-token.md (hunk 2 above)
Comment (cwgoes) [reply packages/specs/pages/base-ledger/fungible-token.md:7]: Ah, I see you do this later, we should either move it up or reference it.

**Review by cwgoes — CHANGES_REQUESTED** [review]

File: packages/specs/pages/interoperability/ibc.md (hunk 3 above)
Comment (yito88) [reply packages/specs/pages/interoperability/ibc.md:28]: Actually, it doesn't have to be an internal address. The current implementation assumes that any balance key's owner is an Address. We could replace it with non-Address. Let me check.

**Comment by cwgoes** [comment]
Is this ready for re-review @yito88?

**Pull request merged**